modules/claims_api/app/controllers/claims_api/v1/forms/intent_to_file_controller.rb

```diff
@@ -20,6 +20,9 @@ def submit_form_0966
           bgs_response = bgs_service.intent_to_file.insert_intent_to_file(intent_to_file_options)
           render json: bgs_response,
                  serializer: ClaimsApi::IntentToFileSerializer
+        rescue Common::Exceptions::Forbidden => e
+          render json: { errors: [{ status: 403, detail: e.errors[0]&.detail }] },
+                 status: :forbidden
```

Comment:
Is this even needed actually? Assuming we keep with Common::Exceptions::Forbidden as the exception, I would assume we're catching and returning a 403 at some point in the stack?

Reply:
Ok, I just tried it out locally and yes, this specific exception is already handled at some point in the stack. Received a 403 status code along with the following response body:
```
{
    "errors": [
        {
            "title": "Forbidden",
            "detail": "Representative cannot file for type 'burial'",
            "code": "403",
            "status": "403"
        }
    ]
}
```
so i dont think we need this extra rescue here, unless we want some other than forbidden of course

Reply:
ahh, ok cool. that makes sense. I'm just adding the custom `detail` attribute and the rest of it should already get caught. will remove 👍